e2e/testsuite/testsuite.go

```diff
@@ -484,6 +491,39 @@ func (s *E2ETestSuite) ExecuteGovProposal(ctx context.Context, chain *cosmos.Cos
 	s.Require().Equal(govtypes.StatusPassed, proposal.Status)
 }
 
+
+// QueryModuleAccountAddress returns the sdk.AccAddress of a given module name.
+func (s *E2ETestSuite) QueryModuleAccountAddress(ctx context.Context, moduleName string, chain *cosmos.CosmosChain) (sdk.AccAddress, error) {
+	authClient := s.GetChainGRCPClients(chain).AuthQueryClient
+
+	moduleAccountsResponse, err := authClient.ModuleAccounts(ctx, &authtypes.QueryModuleAccountsRequest{})
+	if err != nil {
+		return nil, err
+	}
+
+	cfg := simappparams.MakeTestEncodingConfig()
+	authtypes.RegisterInterfaces(cfg.InterfaceRegistry)
```

Comment:
Should we use `cdc := testsuite.Codec()` instead?

Reply:
the only thing here is that this is a `cfg` not a codec. Maybe we can update our helper function to return `cfg, cdc` what do you think?

Reply:
Something like this

```golang
func Codec() *codec.ProtoCodec {
	cdc, _ := codecAndEncodingConfig()
	return cdc
}

func EncodingConfig() simappparams.EncodingConfig {
	_, cfg := codecAndEncodingConfig()
	return cfg
}

func codecAndEncodingConfig() (*codec.ProtoCodec, simappparams.EncodingConfig) {
	cfg := simappparams.MakeTestEncodingConfig()
	banktypes.RegisterInterfaces(cfg.InterfaceRegistry)
	govv1beta1.RegisterInterfaces(cfg.InterfaceRegistry)
	authtypes.RegisterInterfaces(cfg.InterfaceRegistry)
	cdc := codec.NewProtoCodec(cfg.InterfaceRegistry)
	return cdc, cfg
}

```